### DISCUSSION TITLE
The problems when installing ns-3

### DISCUSSION THREAD

**Discussion by DemingCheng** [eclipse/mosaic#117] [Q&A]
![2021-05-29 04-47-49 的屏幕截图](https://user-images.githubusercontent.com/37805861/120042046-f884fa80-c03b-11eb-8757-6e120478cb7c.png)
![2021-05-29 04-47-53 的屏幕截图](https://user-images.githubusercontent.com/37805861/120042053-fa4ebe00-c03b-11eb-85bc-2d98e52fb793.png)
![2021-05-29 04-48-09 的屏幕截图](https://user-images.githubusercontent.com/37805861/120042058-fb7feb00-c03b-11eb-8c3d-98094c858978.png)
Hi, when I use the ns3_installer.sh , there are also only  1829 files in my logs(just as the situation in  https://github.com/eclipse/mosaic/discussions/42#discussioncomment-327715),and there are some "not enabled" info in red ,and "not found" info in yellow in the pictures. Have I installed the ns-3 successfully?

Thank you!

**Comment by erouirior** [comment]
Hi, the above problem might be caused by your pythonxx-dev is missing. If pythonxx-dev is installed, you can try CXXFLAGS="-O3" ./waf --build-profile=optimized --enable-examples --enable-tests configure. Hope this works for you.

**Comment by qw2334** [comment]
Hi, I got the same error message as yours, but the difference is that I have all 1947 files compiled. May I know if you have fixed yours yet?
